src/app/components/Curation/HierarchicalGrid/index.styles.tsx

```diff
@@ -47,6 +47,34 @@ const styles = {
         gridTemplateColumns: 'repeat(4, 1fr)',
       },
     }),
+  singleColumnList: ({ mq }: Theme) =>
+    css({
+      gridTemplateColumns: 'unset',
+
+      [mq.GROUP_3_MIN_WIDTH]: {
+        gridTemplateColumns: 'unset',
+      },
+      [mq.GROUP_4_MIN_WIDTH]: {
+        gridTemplateColumns: 'unset',
+      },
+
+      li: {
+        gridColumn: 'unset',
+        gridRow: 'unset',
+        paddingTop: 0,
+
+        '&::before': {
```

Comment:
What's this one for @amoore108?

Reply:
This is a thin border that is styled on the smaller promos with `::before`, so this will override that style:
<img width="1056" alt="Screenshot 2024-04-23 at 14 50 41" src="https://github.com/bbc/simorgh/assets/11633031/6cab27d9-a1a9-4beb-a5bf-33b5ef6ff1ed">

Reply:
Would border: none be more appropriate CSS in that case?